### PR TITLE
Add link references which were removed with #16011

### LINKED ISSUE #16139: PR #16011 broke links
**This is a Bug Report**
@sftim @makoscafee 

**Problem:**
Pull request broke all links.

**Proposed Solution:**
Revert some changes of #16011 

**Page to Update:**
https://kubernetes.io/docs/tasks/debug-application-cluster/audit

### PR DESCRIPTION
**Pull request by dsbrng25b**
Fix #16139

### CODE REVIEW

**Comment by kbhawkey** [comment]
hello @dsbrng25b. Thanks for this PR.
I do not see the links in the preview page?

Page Preview:
https://deploy-preview-16140--kubernetes-io-master-staging.netlify.com/docs/tasks/debug-application-cluster/audit/

File: content/en/docs/tasks/debug-application-cluster/audit.md [hunk 1]
```diff
@@ -496,4 +496,16 @@ plugin which supports full-text search and analytics.
 
 Visit [Auditing with Falco](/docs/tasks/debug-application-cluster/falco) 
 
+[kube-apiserver]: /docs/admin/kube-apiserver
+[auditing-proposal]: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/auditing.md
```
[the remark this answers is not in the document]
Comment (dsbrng25b) [reply content/en/docs/tasks/debug-application-cluster/audit.md]: The parens were not the problem. The link references has to in the same capture block. Now the links are visible again.

[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/tasks/debug-application-cluster/audit.md]: Nice find :ok_hand:

**Comment by aimeeu** [comment]
I know the main PR has already merged (#16011)....  Falco has its own [documentation](https://falco.org/docs/). Even though this content doesn't exist in the Falco docs, it might be better located there and then linked to from the Kubernetes documentation. I know the Content Guide states that "...Adding content for kubernetes or kubernetes-sigs projects that don’t have their own instructional content" is allowed, so I need to reword and expand that sentence. The end-goal is to keep project documentation in the project's docs so the project's developers know about it and can update it with each release if needed, rather than their having to keep track of project documentation in two places (eg Falco docs and Kubernetes docs).

**Comment by sftim** [comment]
This felt to me like something that falls between two stools: it's about Falco _and_ it's about Kubernetes.
(and that's why I was OK to see it merged)

File: content/en/docs/tasks/debug-application-cluster/audit.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kbhawkey) [reply content/en/docs/tasks/debug-application-cluster/audit.md]: OK.

[the remark this answers is not in the document]
Comment (kbhawkey) [reply content/en/docs/tasks/debug-application-cluster/audit.md]: Let's get this in to fix the links and address the content issue in another PR. Thanks!
/approve

**Pull request merged**